ports.mk

```diff
@@ -21,7 +21,7 @@ install:
 	$(INSTALL) -d "$(DESTDIR)$(datarootdir)/applications"
 	$(INSTALL) -c -m 644 "$(srcdir)/dists/scummvm.desktop" "$(DESTDIR)$(datarootdir)/applications/scummvm.desktop"
 	$(INSTALL) -d "$(DESTDIR)$(datarootdir)/appdata"
```

Comment:
Should this be amended to /metainfo as well?

Reply:
Ups sorry, brown paper bag, fixing in a moment. Thanks for the notice!